### PR TITLE
Feat/merge mobile desktop

### PR DESCRIPTION
**Pull request by sachins-geekyants**
**- What I did**
merged @ mospherePro mobile and desktop code.

### CODE REVIEW

File: lib/view_models/file_transfer_provider.dart [hunk 1]
```diff
@@ -357,10 +359,11 @@ class FileTransferProvider extends BaseModel {
       filename: _filesList[_index].name,
     );
 
-    var _atclient = BackendService.getInstance().atClientInstance;
+    var _atclient = AtClientManager.getInstance().atClient;
     try {
       File file =
-          File(MixedConstants.SENT_FILE_DIRECTORY + _filesList[_index].name);
+          File(MixedConstants.DESKTOP_SENT_DIR + _filesList[_index].name);
```
Comment (sarika01) [inline lib/view_models/file_transfer_provider.dart:365]: Is this right constant?

File: lib/screens/common_widgets/side_bar.dart [hunk 2]
```diff
@@ -119,13 +119,12 @@ class _SideBarWidgetState extends State<SideBarWidget> {
     }
 
     if (contact != null) {
-      if (contact.tags != null && contact.tags['image'] != null) {
-        if (mounted) {
-          setState(() {
-            image = CommonUtilityFunctions().getContactImage(contact);
-          });
-        }
+      if (mounted) {
```
[the remark this answers is not in the document]
Comment (sachins-geekyants) [reply lib/screens/common_widgets/side_bar.dart:122]: the null checks have been moved to `CommonUtilityFunctions().getContactImage` function.

File: lib/view_models/file_transfer_provider.dart (hunk 1 above)
Comment (sachins-geekyants) [reply lib/view_models/file_transfer_provider.dart:365]: yes,sent files are stored in the same directory for mobile and desktop.

File: lib/screens/welcome_screen/widgets/welcome_sceen_home.dart [hunk 3]
```diff
@@ -199,7 +199,7 @@ class _WelcomeScreenHomeState extends State<WelcomeScreenHome> {
                                       child: CommonButton(
                                         TextStrings().buttonResend,
                                         reAttemptSendingFiles,
-                                        buttonColor: Colors.amber[800],
+                                        color: Colors.amber[800],
```
[the remark this answers is not in the document]
Comment (sachins-geekyants) [reply lib/screens/welcome_screen/widgets/welcome_sceen_home.dart:202]: This parameter name was different for mobile and desktop versions.
Now, we are going with color.

File: lib/screens/common_widgets/person_vertical_tile.dart [hunk 4]
```diff
@@ -41,11 +41,10 @@ class _CustomPersonVerticalTileState extends State<CustomPersonVerticalTile> {
 
     // ignore: unnecessary_null_comparison
     if (contact != null) {
-      if (contact.tags != null && contact.tags['image'] != null) {
-        setState(() {
-          image = CommonUtilityFunctions().getContactImage(contact);
-        });
-      }
+      setState(() {
```
[the remark this answers is not in the document]
Comment (sachins-geekyants) [reply lib/screens/common_widgets/person_vertical_tile.dart:44]: The null checks have been moved to `CommonUtilityFunctions().getContactImage` function.

File: lib/screens/history/widgets/received_file_list_tile.dart [hunk 5]
```diff
@@ -100,12 +100,10 @@ class _ReceivedFilesListTileState extends State<ReceivedFilesListTile> {
       contact = checkForCachedContactDetail(widget.receivedHistory.sender);
     }
     if (contact != null) {
-      if (contact.tags != null && contact.tags['image'] != null) {
-        if (mounted) {
-          setState(() {
-            image = CommonUtilityFunctions().getContactImage(contact);
-          });
-        }
+      if (mounted) {
```
[the remark this answers is not in the document]
Comment (sachins-geekyants) [reply lib/screens/history/widgets/received_file_list_tile.dart:103]: The null checks have been moved to `CommonUtilityFunctions().getContactImage` function.

File: lib/main.dart [hunk 6]
```diff
@@ -1,9 +1,17 @@
 import 'package:flutter/material.dart';
 import 'package:at_utils/at_logger.dart';
+import 'package:desktop_window/desktop_window.dart';
 import 'app.dart';
+import 'dart:io';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
   AtSignLogger.root_level = 'finer';
+  if (Platform.isLinux || Platform.isMacOS || Platform.isWindows) {
```
[the remark this answers is not in the document]
Comment (sachins-geekyants) [reply lib/main.dart]: Updated.

**Review by sarika01 — APPROVED** [review]

**Pull request merged**